pkgs/development/libraries/crc32c/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, cmake, gflags }:
+stdenv.mkDerivation rec {
+  pname = "crc32c";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "crc32c";
+    rev = version;
+    sha256 = "1sazkis9rzbrklfrvk7jn1mqywnq4yghmzg94mxd153h8b1sb149";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ gflags ];
+
```

```suggestion
NIX_CFLAGS_COMPILE = stdenv.lib.optional stdenv.isAarch64 "-march=armv8-a+crc";

```
I just saw that the build fails on aarch64-linux and recognized the error from #65504.
Building this without crc extensions fails on arm. 
Setting the target architecture as above should fix that issue and maybe the platforms can even be restricted a bit more.
Otherwise this looks fine to me :+1:.